src/Entity/BlockManager.php

```diff
@@ -43,8 +40,6 @@ public function save($block, $andFlush = true)
      * @param int  $parentId Parent block Id (needed when partial = true)
      * @param int  $pageId   Page Id (needed when partial = true)
      * @param bool $partial  Should we use partial references? (Better for performance, but can lead to query issues.)
-     *
-     * @return mixed
```

Comment:
If we will remove the `mixed` types, we should update the rule at [dev-kit](https://github.com/sonata-project/dev-kit/blob/165dd7bb80c3e3a6ce69f0f8f440dd737ea25a47/project/.php_cs.dist.twig#L31).

Reply:
IMHO we could leave the settings as they are. It is no problem to keep some mixin settings. I reverted some of my changes.